craft.lic

```diff
@@ -0,0 +1,291 @@
+custom_require.call(%w[drinfomon equipmanager common common-crafting common-items common-money common-travel])
```

Comment:
💡 Please add a header with a link to elanthipedia

Reply:
done